quality/postman/emodb_tests.postman_collection.json

```diff
@@ -40468,8 +40468,9 @@
 															"});",
 															"",
 															"pm.test(\"Body matches string\", function () {",
-															"    pm.expect(pm.response.json()).to.eql({\"reason\":\"not authorized\"});",
-															"});"
+															"    pm.expect(pm.response.json().reason).to.be.oneOf([`not authorized to update table ${pm.environment.get('table')}`, 'not authorized']);",
```

Comment:
it's possible to have 2 different outputs?

Reply:
Yes it is, if I add if else statement to verify which environment tests are run on because on local there is one response and on cert/ci more prod like environment we return a bit different response text. I decided to go with built in postman/chai solution to use to.be.oneOf asserter as for this particular case I believe it is fine. What do you say, we can proceed like this or you think we should add if else statement for checking environment for this kind of situations?

Reply:
we can proceed with this as temporary solution, but it's weird that tests behave differently depend on env. please create ticket to investigate and fix that.

Reply:
EMO-7638 has been created for this